keyboards/bm16s/keymaps/media/keymap.c

```diff
@@ -0,0 +1,74 @@
+#include "bm16s.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```